src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    chipVersion: version,
+    get info() {
+      return `name: ${this.name}, chip version: `
+      + `${this.chipVersion}, wheels: ${this.wheels}`;
+    },
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goRight(steps = 1) {
+      if (steps < 0) {
+        return this;
+      }
+      this.coords.x += steps;
```

Comment:
Интересный реверс, но так будет покороче:
```suggestion
      if (steps > 0) {
        this.coords.x += steps;
      }
      
```